src/main/java/kr/motd/maven/os/Detector.java

```diff
@@ -245,33 +257,33 @@ private static String normalize(String value) {
         return value.toLowerCase(Locale.US).replaceAll("[^a-z0-9]+", "");
     }
 
-    private static LinuxRelease getLinuxRelease() {
+    private static LinuxRelease getLinuxRelease(FileActionFacade fileActionFacade) {
```

Comment:
Since this and similar are private, it may make sense to make them non-static instead of passing in the providers.

Reply:
Yeah. I was unsure about `determineBitness()` as it was public (I turned it to private and thought it would be easy to flip it back if they were intended to be public APIs). 

Anyway, now I changed those static utilities to no non-static helpers. @trustin Please let me know if the change to `determineBitness()` is broken.